### PR TITLE
Feature/33 fe create login page

### PR DESCRIPTION
**Pull request by mulanga-sibeli-retrorabbit**

### CODE REVIEW

**Review by Jumanahalhazba — APPROVED** [review]

File: ChatGPT-Bot/package.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
     "@angular/platform-browser": "^15.0.0",
     "@angular/platform-browser-dynamic": "^15.0.0",
     "@angular/router": "^15.0.0",
+
```
Comment (De-Hann) [inline ChatGPT-Bot/package.json:21]: empty line

File: ChatGPT-Bot/package.json [hunk 2]
```diff
@@ -18,7 +18,9 @@
...
     "primeicons": "^6.0.1",
+
```
Comment (De-Hann) [inline ChatGPT-Bot/package.json:23]: empty line

File: ChatGPT-Bot/src/app/app.component.html [hunk 3]
```diff
@@ -1 +1,7 @@
+
+<style></style>
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/app.component.html:2]: Why are there empty style tags?

File: ChatGPT-Bot/src/app/app.component.html [hunk 4]
```diff
@@ -1 +1,7 @@
...
+
+<app-login-page></app-login-page>
+
+
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/app.component.html:6]: Why is there so much empty space

File: ChatGPT-Bot/src/app/app.module.ts [hunk 5]
```diff
@@ -4,18 +4,22 @@ import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
...
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+
 import { CoreModule } from './core/core.module';
 import { PrimeModule } from './shared/assets/prime.module';
 import { AvatarUserComponent } from './shared/components/avatar-user/avatar-user.component';
 import { ChatGPTComponent } from './shared/pages/chat-gpt/chat-gpt.component';
 import { AvatarBotComponent } from './shared/components/avatar-bot/avatar-bot.component';
+import { LoginPageComponent } from './pages/login-page/login-page.component';
+
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/app.module.ts:14]: empty space

File: ChatGPT-Bot/src/app/pages/login-page/login-page.component.scss [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    .google-icon{
+        z-index: 1000;
+    }
+
+    .main-container{
+        background: rgb(93,58,133);
+        background: linear-gradient(90deg, rgba(93,58,133, 0.891) 0%, rgba(74,151,155,1) 100%);
+        width: 100vw;;
+        height: 100vh;
+    }
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/pages/login-page/login-page.component.scss:41]: Main container is specified twice

**Comment by De-Hann** [comment]
Remove unused image

File: ChatGPT-Bot/src/app/pages/login-page/login-page.component.scss (hunk 6 above)
Comment (De-Hann) [inline ChatGPT-Bot/src/app/pages/login-page/login-page.component.scss:41]: Remove empty space

File: ChatGPT-Bot/src/app/shared/components/avatar-bot/avatar-bot.component.spec.ts [hunk 7]
```diff
@@ -8,9 +8,8 @@ describe('AvatarBotComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ AvatarBotComponent ]
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/shared/components/avatar-bot/avatar-bot.component.spec.ts:11]: remove spec.ts

File: ChatGPT-Bot/src/app/shared/components/avatar-user/avatar-user.component.spec.ts [hunk 8]
```diff
@@ -8,9 +8,8 @@ describe('AvatarUserComponent', () => {
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ AvatarUserComponent ]
-    })
-    .compileComponents();
```
Comment (De-Hann) [inline ChatGPT-Bot/src/app/shared/components/avatar-user/avatar-user.component.spec.ts:13]: remove spec.ts

**Review by De-Hann — APPROVED** [review]

**Pull request merged**